code/modules/mob/living/carbon/alien/special/alien_embryo.dm

```diff
@@ -65,7 +65,7 @@
 
 
 
-/obj/item/organ/internal/body_egg/alien_embryo/proc/AttemptGrow(gib_on_success = 1)
+/obj/item/organ/internal/body_egg/alien_embryo/proc/AttemptGrow(burst_on_success = 1)
 	if(!owner || polling)
 		return
 	polling = 1
```

Comment:
Just for simple code readability:

```suggestion
	polling = TRUE
```

Please change `polling = 0` to `FALSE` later, GitHub doesn't let me comment on it.

Reply:
Done!